.vscode/settings.json

```diff
@@ -24,4 +24,13 @@
         "editor.tabSize": 2,
         "editor.autoIndent": "advanced"
     },
+    "python.testing.unittestArgs": [
+        "-v",
+        "-s",
+        "./tests",
+        "-p",
+        "test_*.py"
+    ],
+    "python.testing.pytestEnabled": false,
+    "python.testing.unittestEnabled": true,
```

Comment:
Can you remove these? We should only specify one set of test configuration using the `tox` configuration in `pyproject.toml` to prevent any confusion, and be IDE / editor agnostic :)